src/ReactLeafletMap/layers/WmsLayer/WMSTileLayerWithFetchedTiles/index.js

```diff
@@ -0,0 +1,76 @@
+import {
+	createTileLayerComponent,
+	updateGridLayer,
+	withPane,
+} from '@react-leaflet/core';
+import {TileLayer} from 'leaflet';
+
+async function fetchImage(url, callback, headers, abort) {
+	const controller = new AbortController();
+	const signal = controller.signal;
+	if (abort) {
+		abort.subscribe(() => {
+			controller.abort();
+		});
+	}
+	const f = await fetch(url, {
+		method: 'GET',
+		headers: headers,
+		mode: 'cors',
+		signal: signal,
+	});
+	const blob = await f.blob();
+	callback(blob);
+}
+
+var LWMSTileLayerWithFetchedTiles = TileLayer.WMS.extend({
```

Comment:
What does the L in LWMSTile... mean? And why not const?

Reply:
Probably "Leaflet", it is copypasted. I have published updated version